app/uk/gov/hmrc/emailverification/models/PasscodeDoc.scala

```diff
@@ -17,12 +17,18 @@
 package uk.gov.hmrc.emailverification.models
 
 import org.joda.time.DateTime
-import play.api.libs.json.{Format, Json, OFormat}
+import play.api.libs.json.{Format, JsObject, JsResult, JsValue, Json, OFormat}
 import uk.gov.hmrc.mongo.json.ReactiveMongoFormats
 
-case class PasscodeDoc(sessionId:String, email: String, passcode: String, expireAt: DateTime)
+case class PasscodeDoc(sessionId:String, email: String, passcode: String, expireAt: DateTime, passcodeAttempts:Int=0, emailAttempts:Int=0)
```

Comment:
is any of this code in production? default parameters are ignored when deserialising JSON

Reply:
no and not much point in these defaults actually

Reply:
No, don't need these defaults actually